From 4ba789d827d888bacdaddad129108eb481211e3c Mon Sep 17 00:00:00 2001
From: Nathan Hakkakzadeh <nhakkakzadeh@mozilla.com>
Date: Thu, 2 Jun 2016 13:19:28 -0700
Subject: [PATCH 05/12] Bug 1100925 - Added flags in virtualenv to
 differenciate between a native Windows version of Python and a MinGW64
 version of Python running in msys2. r=gps

This is going to be necessary in the following commits to ensure that virtualenv has access to native libraries.

MozReview-Commit-ID: FwxOoCWnQMW

--HG--
extra : rebase_source : c4cf829231757b3dc2c5ca821c7248dc3fb0d763

(cherry picked from commit 57e72ee336754d5cabb9f53cb5dfca7f817c6a18)
Signed-off-by: Jacek Caban <jacek@codeweavers.com>
---
 python/virtualenv/virtualenv.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/python/virtualenv/virtualenv.py b/python/virtualenv/virtualenv.py
index 13146f66eaaf..a1854ab36465 100755
--- a/python/virtualenv/virtualenv.py
+++ b/python/virtualenv/virtualenv.py
@@ -53,8 +53,9 @@ py_version = 'python%s.%s' % (sys.version_info[0], sys.version_info[1])
 
 is_jython = sys.platform.startswith('java')
 is_pypy = hasattr(sys, 'pypy_version_info')
-is_win = (sys.platform == 'win32')
+is_win = (sys.platform == 'win32' and os.sep == '\\')
 is_cygwin = (sys.platform == 'cygwin')
+is_msys2 = (sys.platform == 'win32' and os.sep == '/')
 is_darwin = (sys.platform == 'darwin')
 abiflags = getattr(sys, 'abiflags', '')
 
-- 
2.24.1

